modules/machineset-yaml-vsphere.adoc

```diff
@@ -75,7 +75,14 @@ $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 <2> Specify the infrastructure ID and node label.
 <3> Specify the node label to add.
 <4> Specify the vSphere VM network to deploy the machine set to.
-<5> Specify the vSphere VM template to use, such as `user-5ddjd-rhcos`.
+<5> Specify the vSphere VM clone of the template to use, such as `user-5ddjd-rhcos`.
++
+[IMPORTANT]
+====
+Do not specify the original VM template. The VM template must remain off and must be cloned for new {op-system} machines. Starting the VM template configures the VM template as a VM on the platform, which prevents it from being used as a template that machine sets can apply configurations to.
+//This admonition note also appears in `modules/installation-vsphere-machines.adoc` and `modules/windows-machineset-vsphere.adoc`.
```

Comment:
Same concern here with the template remaining off. Not sure what that means. Might be apparent to the audience, though.

Reply:
Going to leave this alone as well. I believe there is a "Power ON" or similar step one would have to take to actually start the VM. Point being that the user should not start the template and should only start a clone(s) of the template.